hadoop-ozone/common/src/main/java/org/apache/hadoop/ozone/om/helpers/ServiceInfoEx.java

```diff
@@ -18,19 +18,26 @@
 
 package org.apache.hadoop.ozone.om.helpers;
 
+import org.apache.hadoop.hdds.scm.client.ClientTrustManager.CACertificateProvider;
```

Comment:
It's good practice to reference an class from a client module in a common module.